nginx/html/index.html

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
This html contains references to third-party tutorial. Please make it empty "hello nginx" html. Also, remove .swp file from version control.